dist/js/shepherd.js

```diff
@@ -207,18 +206,23 @@ var Step = (function (_Evented) {
   }, {
     key: 'getAttachTo',
     value: function getAttachTo() {
+      var returnOpts = {};
       var opts = parsePosition(this.options.attachTo) || {};
-      var selector = opts.element;
-
-      if (typeof selector === 'string') {
-        opts.element = document.querySelector(selector);
+      returnOpts.on = opts.on;
+      if (typeof opts.offset !== 'undefined') {
+        returnOpts.offset = opts.offset;
+      }
```

Comment:
You're right - I got this confused with something else, it's only a single level of props so i wouldn't need deep copy anyway.